### PR TITLE
#5344 Event log unread count is not cleared

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts [hunk 1]
```diff
@@ -86,5 +87,38 @@ describe('App Bar', () => {
       // The event log message should be cleared by now
       cy.findByText('No log').should('exist');
     });
+
+    it('should show the notifications when creating pipeline failed', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/eventLog.test.ts:91]: _a pipeline_ :)

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts [hunk 2]
```diff
@@ -86,5 +87,38 @@ describe('App Bar', () => {
...
+      // create a pipeline by native api
+      cy.request('POST', 'api/pipelines', {
+        name: 'p1',
+        // this is the hash value of "workspace1 + workspace"
+        group: '3e1117d3c',
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/eventLog.test.ts:96]: Why not using hashByGroupAndName here 🤔

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts [hunk 3]
```diff
@@ -86,5 +87,38 @@ describe('App Bar', () => {
...
+      });
+
+      cy.createPipeline('p1');
+      cy.findByTitle('Event logs').within(() => {
+        //This is the notification; should be one error
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/eventLog.test.ts:101]: _should have_

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/client/cypress/it/appBar/eventLog.test.ts:96]: Not sure why I'm not using it...🙄

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts [hunk 4]
```diff
@@ -86,5 +87,38 @@ describe('App Bar', () => {
...
+        cy.contains('span', 2).should('exist');
+      });
+
+      // open event log will clear the notifications
+      cy.findByTitle('Event logs').click();
+      cy.findByTitle('Event logs').within(() => {
+        cy.contains('span', 2).should('not.exist');
+      });
+
+      // when opening event log, the notification should not appear
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/eventLog.test.ts:117]: You mean the notification count should be gone here, right?

Comment (saivirtue) [reply ohara-manager/client/cypress/it/appBar/eventLog.test.ts:117]: Yes :)

File: ohara-manager/client/src/store/epics/eventLog/updateNotificationsEpic.js [hunk 5]
```diff
@@ -48,8 +48,9 @@ const increaseNotification$ = (entity, state$) => {
 
 export default (action$, state$) =>
   action$.pipe(
-    // we listen the create event epic
-    ofType(actions.createEventLog.SUCCESS, actions.updateNotifications.TRIGGER),
+    ofType(actions.updateNotifications.TRIGGER),
+    // update notifications if and only if isOpen=false
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/eventLog/updateNotificationsEpic.js:52]: 👍

File: ohara-manager/client/cypress/it/appBar/eventLog.test.ts (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/cypress/it/appBar/eventLog.test.ts]: XXD

File: ohara-manager/client/cypress/it/navigator/settings.test.ts [hunk 6]
```diff
@@ -672,7 +672,7 @@ describe('Navigator', () => {
     });
   });
 
-  context.only('Files Settings', () => {
+  context('Files Settings', () => {
```
Comment (saivirtue) [inline ohara-manager/client/cypress/it/navigator/settings.test.ts:675]: My bad of last commit...

**Pull request merged**